src/nvim/main.c

```diff
@@ -207,6 +207,7 @@ int main(int argc, char **argv)
   // Get the name with which Nvim was invoked, with and without path.
   set_vim_var_string(VV_PROGPATH, (char_u *)argv[0], -1);
   set_vim_var_string(VV_PROGNAME, path_tail((char_u *)argv[0]), -1);
+  set_vim_var_dict(VV_COMPLETED_ITEM, dict_alloc());
```

Comment:
Are you sure this is needed here? This already seems to be dealt with in `edit.c`, and `v:completion_item` seems to work fine without it.